widget/embedded/src/store/quote.ts

```diff
@@ -29,6 +34,21 @@ export const getUsdValue = (
 export type Meta = Pick<MetaResponse, 'blockchains' | 'tokens'>;
 
 export type SetTokenParams = { token: Token; meta: Meta } | { token: null };
+export type SelectedQuote = {
+  swaps: SwapResult[];
+  requestId: string;
+  outputAmount?: string;
+  resultType?: RoutingResultType;
+  tags?: RouteTag[];
+  validationStatus: BlockchainValidationStatus[] | null;
+  requestAmount: string;
+};
```

Comment:
Could you relocate this type to 'widget/embedded/src/types/quote.ts'?

Reply:
yes, i could. fixed